Utils/src/CandidateTrackFilter.cc

```diff
@@ -0,0 +1,366 @@
+//-----------------------------------------------------------------------------------------
```

Comment:
this large totally irrelevant block comment can be removed

Reply:
Done